app/api/crud.py

```diff
@@ -195,13 +195,53 @@ async def get(
                             "subject_group": "first",
                             "assessment": lambda x: list(x.unique()),
                             "image_modal": lambda x: list(x.unique()),
-                            "pipeline_version": lambda x: list(x.unique()),
-                            "pipeline_name": lambda x: list(x.unique()),
                             "session_file_path": "first",
                         }
                     )
                 )
 
+                pipeline_data = (
+                    group.groupby(
+                        [
+                            "sub_id",
+                            "session_id",
+                            "session_type",
+                            "pipeline_name",
+                        ]
+                    )
+                    .agg(
+                        {
+                            "pipeline_version": lambda x: list(
```

Comment:
Just to double check: assuming that all pipeline version numbers are stored in JSONLDs as strings, we would always expect the `pipeline_version` column in the graph result to contain only strings, correct?

Reply:
Yes